variables.tf

```diff
@@ -63,6 +63,7 @@ variable "worker_pools" {
       kms_account_id  = optional(string)
     }))
     additional_security_group_ids = optional(list(string))
+    import_on_create              = optional(bool) # Import an existing WorkerPool from the cluster, instead of creating a new.
```

Comment:
I do not think this should be exposed here to the consumer. I think the approach should be that import_on_create is automatically set to true by the module for the default pool (and not for the other), with an option, at the root level to avoid this behavior.